### PR TITLE
Patents fr.md

### PR DESCRIPTION
**Pull request by Tomahna81**
This is a translation of the file patents_en.md that can be found at: https://github.com/monero-ecosystem/outreach-docs/delete/master/monero-outreach-docs/en/storyboard/patents_en.md

### CODE REVIEW

File: monero-outreach-docs/translations/fr/storyboard/patents_fr.md
Comment (Hsterofemia) [inline monero-outreach-docs/translations/fr/storyboard/patents_fr.md:74]: - brevets ont déjà été découverte -> brevets ont déjà été découvertes ; "découvertes" s'accorde avec "technologies"
- jamais dû être brevetée -> jamais dû être brevetées ; "brevetées" s'accorde avec "technologies"
- procédure de réexamen entamé -> procédure de réexamen entamée; "entamée" s'accorde avec "procédure"
- que la communauté ai -> que la communauté ait ; "qu'il ait", subjonctif présent

**Review by Tomahna81 — COMMENTED** [review]
Thank you @Hsterofemia for this review. I basically followed all your advices and applied all of your corrections. The only thing I could not change concerns the bibliography number 8. This because even in the original, the link to it is missing. Perhaps @lh1008 can do something about this?

**Review by Hsterofemia — APPROVED** [review]
The changes were indeed integrated.

**Review by Hsterofemia — APPROVED** [review]
The changes were indeed integrated.

**Review by Hsterofemia — COMMENTED** [review]
The changes were indeed integrated.

**Review by Hsterofemia — COMMENTED** [review]
The changes were indeed integrated.

**Review by Hsterofemia — APPROVED** [review]
The changes were indeed integrated.

**Comment by Hsterofemia** [comment]
The review comments were correctly integrated, so this commit has the reviewer's approval. It was spread throughout 5 commits though, so I'm not sure how we proceed. I approved all 5, @lh1008 let me know if you need anything else from my end to close this PR. Great work everyone!

**Comment by lh1008** [comment]
Okay, this should be ready to merge. @Tomahna81 could you please squash the commits so we can merge it with only one commit for the whole PR?
Let me know if you need help with this.

**Comment by Tomahna81** [comment]
Closing this pull request: Due to a wrong manipulation, it cannot be merged as it is. A new one will be opened with the latest version of patents_fr.md.

**Pull request closed without merging**
